docs/src/administration/web/_index.md

```diff
@@ -116,7 +116,37 @@ The URLs pull-down exposes the domains that can be used to access application en
 
 #### GIT
 
-The Git pull-down displays the commands to use to clone the codebase via Git.
+The Git pull-down displays the commands to use to clone the codebase via Git. If you are using Platform.sh as your primary remote repository, the command shown will clone from the project. Otherwise if you have set up an [external integration](/integrations/source/_index.md), the command will clone directly from the integrated remote repository, which is considered the [source of truth](/administration/users.md) for the project.
+
+{{< note >}}
```

Comment:
Making this whole thing a note seems excessive.  It's not an aside, but a non-trivial part of the process.

Reply:
I get you. I started here, then grew out `users.md`, which in retrospect seems more like what all of these should link to. Maybe I'll nix the note for a link here and move all this to `users`.